tests/snmpsim/zynos-mes.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|MES Something
```

Comment:
I have copied and modified the original file zynos-es.snmprec

`1.3.6.1.2.1.1.1.0|4|ES Something`
`1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.890`

I thought that's how I'll support all switches ZyXEL MES series.